Makefile

```diff
@@ -0,0 +1,8 @@
+help: ## Print this help
```

Comment:
What are the advantages of the Makefile?

Reply:
I use a lot to provide simple tasks independently of the build tool:
```
make build
``` 
builds the project and do all the magic stuff I don't to remember per project, and it's the only command I need to remember. It provides a simple contract.
I can delete it if you prefer.